### ISSUE TITLE
Failing test on whisper 

### ISSUE THREAD

**Issue by xymz** [ml-explore/mlx-examples#395]
```
(env) {24-01-31 21:53}:~/projects/mlx-examples/whisper@main✗✗✗✗✗✗ user% python test.py
/Users/user/projects/mlx-examples/env/lib/python3.9/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
.........F.
======================================================================
FAIL: test_transcribe_alice (__main__.TestWhisper)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/user/projects/mlx-examples/whisper/test.py", line 312, in test_transcribe_alice
    check_segment(result["segments"][73], expected_73)
  File "/Users/user/projects/mlx-examples/whisper/test.py", line 306, in check_segment
    self.assertAlmostEqual(seg[k], v, places=3)
AssertionError: -0.13381177127951444 != -0.1350895343440594 within 3 places (0.0012777630645449678 difference)

----------------------------------------------------------------------
Ran 11 tests in 14.997s

FAILED (failures=1)
```

FYI, my environment is `Apple M3 Pro` and MacOS Sonoma 14.2.1(23C71). I didn't run with this before other platform(e.g m1), so I can't ensure that it is my environment problem or not. But other tests are working well.

**Comment by awni** [comment]
Yea I'm not sure why it started failing. Fix is in #393

**Comment by xymz** [comment]
Amazing. Thanks for fast patch.
